frameworks/C++/wt/setup_postgres.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
+export BOOST_ROOT=/usr/local
+export BOOST_INC=${BOOST_ROOT}/include
+export BOOST_LIB=${BOOST_ROOT}/lib
+
+export WT_ROOT=${IROOT}/wt
+export WT_LIB=${WT_ROOT}/lib
+export WT_INC=${WT_ROOT}/include
 
-# This has to be included here since it is one of the vars NOT copied from the original environment
```

Comment:
Please add this comment back - it is important to note that certain special variables are not visible here (such as `PATH`, `LD_LIBRARY_PATH`, etc)